src/app/[locale]/dataset-card.tsx

```diff
@@ -11,6 +11,7 @@ export default function DatasetCard({dataset, locale}: Props) {
     <div
       key={dataset.id}
       className="group relative flex flex-col overflow-hidden rounded-lg border border-gray-200 bg-white"
+      data-test="dataset-card"
```

Comment:
Curious: is there a way to remove the `data-test` attributes when creating a production build?

Reply:
Yes, but something to keep in mind, if we do this, we cannot do end-to-end tests on production environments. 

You can do this with https://nextjs.org/docs/advanced-features/compiler#remove-react-properties

// next.config.js
module.exports = {
  compiler: {
    reactRemoveProperties: process.env.NODE_ENV === "production",
  },
}

Reply:
Good to know - thank you!